client/my-sites/importer/author-mapping-item.jsx

```diff
@@ -41,11 +42,10 @@ export default class extends React.PureComponent {
 		}
 	}
 
-	getCurrentUser = () => {
-		const currentUser = user().get();
-
-		return Object.assign( {}, currentUser, { name: currentUser.display_name } );
-	};
+	getCurrentUser = () => ( {
+		...this.props.currentUser,
+		name: this.props.currentUser.display_name,
```

Comment:
This method could be merged into the `mapStateToProps` call.

Or, even better, what if we changed the `components/user` and `components/gravatar` to support both `display_name` and `name` properties? Then we wouldn't need to assign `name` here, and `Gravatar` would start showing correct `alt` image descriptions for `user` objects that only have `name`.

Reply:
done

Reply:
dug into this a bit more.

`/me` gives back an object with `display_name`, but not `name`.
`/sites/:site/users` gives back objects with `name` and `nice_name`, but no `display_name`.

feels likes we should should be normalizing user objects somewhere.